vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/ByLocatorSearch.java

```diff
@@ -31,6 +31,14 @@ public ByLocatorSearch(LocatorType type)
     @Override
     public List<WebElement> search(SearchContext searchContext, SearchParameters parameters)
     {
-        return findElements(searchContext, getType().buildBy(parameters.getValue()), parameters);
+        try
```

Comment:
I don't think we need to handle runtime exceptions in order to find a way how to create a locator by its type, it is static data and we already know what support locator type

Reply:
> 

But we don't know what exactly type it will be
And, accordingly, what it supports and what does not